Dockerfile.prod

```diff
@@ -0,0 +1,27 @@
+# syntax=docker/dockerfile:1
+ARG PYTHON_VERSION=3.11
```

Comment:
We have a small issue here:  .prod is too generic because this is .prod for the Volto frontend, but we need to name a second Dockerfilehere that generates the prod-classicui server image.

Add comment:

This is a basic production image for the Plone 'content api' backend that is required for the react based  plone-frontend  image (Volto)